builder/comp-builder.nix

```diff
@@ -14,6 +14,7 @@ let self =
 
 , preUnpack ? component.preUnpack, postUnpack ? component.postUnpack
 , configureFlags ? component.configureFlags
+, prePatch ? component.prePatch, postPatch ? component.postPatch
```

Comment:
I think I had the same (or similar in the cross branch?) but yea, this seems sensible.